tests/eko/test_ad.py

```diff
@@ -54,42 +55,44 @@ def test_eigensystem_gamma_singlet_projectors_EV():
 
 def test_gamma_ns():
     nf = 3
+    p = False
```

Comment:
As already noticed in #184 these tests contains a lot of repetitions. We want to refactor limiting the amount of repetitions as you would for normal code.

E.g. here you can do it by looping over orders and outputs, without repeating very similar calls over and over.